integration/simulation/simulation.go

```diff
@@ -53,15 +55,22 @@ func NewSimulation(nrNodes int, l1NetworkCfg *L1NetworkCfg, l2NetworkCfg *L2Netw
 
 		// create an enclave server
 		nodeID := common.BigToAddress(big.NewInt(int64(i)))
-		port := uint64(ENCLAVE_CONN_START_PORT + i)
-		server, err := rpc.StartServer(port, nodeID, stats)
-		if err != nil {
-			panic(fmt.Sprintf("failed to create enclave server: %v", err))
+		var enclaveClient enclave.Enclave
+		if virtualEnclave {
+			enclaveClient = rpc.NewEnclaveFakeRPCClient(nodeID, stats)
```

Comment:
Similarly, if we're keeping the fake RPC client (see above), perhaps name it `NewEnclaveLocalClient` or similar instead?

Reply:
Yep changing it to `EnclaveTestRPCClient` and `NewEnclaveTestRPCClient` respectively.